src/outfit.c

```diff
@@ -1830,53 +1828,56 @@ static void outfit_parseSLauncher( Outfit* temp, const xmlNodePtr parent )
-   SDESC_COND( l, temp, _("\n%.2f DPS [%.0f Damage]"),
-         1. / temp->u.lau.delay * temp->u.lau.dmg.damage * (double)temp->u.lau.shots, temp->u.lau.dmg.damage * (double)temp->u.lau.shots );
-   SDESC_COND( l, temp, _("\n%.1f Disable/s [%.0f Disable]"),
-         1. / temp->u.lau.delay * temp->u.lau.dmg.disable * (double)temp->u.lau.shots, temp->u.lau.dmg.disable * (double)temp->u.lau.shots );
-   SDESC_COND( l, temp, _("\n%.0f Hit Radius%s"), temp->u.lau.radius, (outfit_isProp(temp,OUTFIT_PROP_WEAP_FRIENDLYFIRE)?"!!":"") );
-   SDESC_ADD(  l, temp, _("\n%.1f Shots Per Second"), 1. / temp->u.lau.delay );
-   SDESC_ADD(  l, temp, _("\n%s Range [%.1f duration]"), num2strU( outfit_range(temp), 0 ), temp->u.lau.duration );
+   SDESC_ADD(  l, temp, _("\n  Holds %d ammo"), temp->u.lau.amount );
+   SDESC_ADD(  l, temp, "\n  %s %.0f%%", _("Penetration:"), temp->u.lau.dmg.penetration * 100. );
+   SDESC_COND( l, temp, "\n  %s %.2f [%.0f /sec]", _("Damages:"),
+         temp->u.lau.dmg.damage * (double)temp->u.lau.shots,
+         1. / temp->u.lau.delay * temp->u.lau.dmg.damage * (double)temp->u.lau.shots);
+   SDESC_COND( l, temp, "\n  %s %.1f [%.0f /sec]", _("Disable:"),
+         temp->u.lau.dmg.disable * (double)temp->u.lau.shots,
+         1. / temp->u.lau.delay * temp->u.lau.dmg.disable * (double)temp->u.lau.shots);
+   SDESC_COND( l, temp, "\n  %s %.0f %s", _("Hit Radius:"),
+          temp->u.lau.radius,
+          (outfit_isProp(temp,OUTFIT_PROP_WEAP_FRIENDLYFIRE)?"#r!!#0":""));
+   SDESC_ADD(  l, temp, "\n  %s %.1f /sec", _("Shots:"), 1. / temp->u.lau.delay );
+   SDESC_ADD(  l, temp, "\n  %s %s [%.1f duration]", _("Range:"), num2strU( outfit_range(temp), 0 ), temp->u.lau.duration );
    if (temp->u.lau.thrust > 0.) {
       if (temp->u.lau.speed > 0.)
-         SDESC_ADD( l, temp, _("\n%.0f Initial Speed (%.0f Thrust)"), temp->u.lau.speed, temp->u.lau.thrust );
-      else
-         SDESC_ADD( l, temp, _("\n%.0f Thrust"), temp->u.lau.thrust );
+         SDESC_ADD( l, temp, "\n  %s %.0f", _("Initial Speed:"), temp->u.lau.speed);
+      SDESC_ADD( l, temp, "\n  %s %.0f", _("Thrust:"), temp->u.lau.thrust );
    }
    else
-      SDESC_COND( l, temp, _("\n%.0f Speed"), temp->u.lau.speed );
+      SDESC_COND( l, temp, "\n  %s %.0f", _("Initial Speed:"), temp->u.lau.speed );
    if (!(temp->u.lau.thrust > 0. && temp->u.lau.speed > 0.))
-      SDESC_ADD(  l, temp, _("\n%.0f Maximum Speed"), temp->u.lau.speed_max );
-   SDESC_ADD(  l, temp, _("\n%.1f Seconds to Reload"), temp->u.lau.reload_time );
-   SDESC_COND( l, temp, _("\n%.1f EPS [%.0f Energy]"), temp->u.lau.delay * temp->u.lau.energy, temp->u.lau.energy );
-   SDESC_COND( l, temp, _("\n%.1f MJ Armour [%.0f%%]"), temp->u.lau.armour, temp->u.lau.dmg_absorb*100.);
-   SDESC_COND( l, temp, _("\n%.0f%% Jam Resistance"), temp->u.lau.resist * 100. );
+      SDESC_ADD(  l, temp, "\n  %s %.0f", _("Maximum Speed:"), temp->u.lau.speed_max );
```

Comment:
I inversed how it was originaly done. 
Instead of space only the armour and shield damages bonus i keeped them aligned and spaces de others.
I guess there should be major and minor stats and handle order and spacing according. I'll dig into statistics to classify them.